### PR TITLE
Removed crv from old staking, useless comparred to used gas

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: contracts/staking/StakingV1_1.sol [hunk 1]
```diff
@@ -116,22 +84,10 @@ contract StakingV1_1 is StakingState, StakingConstants, PausableGuardian {
         }
     }
 
-
-    function _depositTo3Pool(
-        uint256 amount)
-        internal
+    function withdrawFrom3Pool(uint256 amount)
+        public onlyOwner
```
Comment (RomanHiden) [inline contracts/staking/StakingV1_1.sol]: shall we make it onlyGuardian?

Comment (Drypto13) [reply contracts/staking/StakingV1_1.sol]: I would make this onlyGuardian

**Review by Drypto13 — COMMENTED** [review]
Looks good. I would recommend that onlyGuardian is used as mentioned above but everything else is fine.

File: contracts/staking/StakingV1_1.sol (hunk 1 above)
Comment (RomanHiden) [reply contracts/staking/StakingV1_1.sol]: @eugenesh1 make it guardian

**Comment by RomanHiden** [comment]
please add some tests as well

**Review by Drypto13 — APPROVED** [review]

**Review by RomanHiden — APPROVED** [review]
looks good

**Pull request merged**